modules/installation-vsphere-machines.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere.adoc
+
+[id="installation-vsphere-machines-{context}"]
+= Creating {op-system-first} machines in vSphere
+
+Before you install a user provisioned infrastructure (UPI) cluster on vSphere,
+you need to create {op-system} machines on a single host for it to use.
+
+.Prerequisites
+
+* Obtain the Ignition config files for your cluster.
+* Have access to an HTTP server that you can access from your computer and that
+the machines that you create can access.
+
+.Procedure
```

Comment:
Does anyone really try such procedure step by step strictly. 
Today I tried the whole day, never had a successful install. I am using vcsa-qe.vmware.devcluster.openshift.com cluster for testing. Seem like bootstrap never get ready, I can not login bootstrap either, I have no way to debug where is wrong? I am not sure it is caused by missed some steps in this doc, or my understanding for the procedure is wrong. Anyone could provide help here? If anyone want to check my cluster, log in https://vcsa-qe.vmware.devcluster.openshift.com, and switch to "VMs and Templates inventory view", check 'jialiu-vsphere1' folder for all my instances. My validation is totally is blocked here.

Reply:
I also opened a bug for that:
https://bugzilla.redhat.com/show_bug.cgi?id=1703803

Reply:
https://github.com/openshift/openshift-docs/pull/14523#discussion_r279456130

Reply:
Sorry, my notes said "Host the bootstrap after you base64 encode it." I've updated the docs to not base64 encode the bootstrap.ign file.

Reply:
Tried, still not working. leave comments in bugzillar.
I have one question, is there such similar setting (e.g: guestinfo.ignition.config.data:) when deploying cluster via Terraform? I did not see such setting in a cluster deployed via Terraform. Do I miss something? If indeed no such setting, how to inject ignition files into instances when launching via Terraform. Any hint?

Reply:
The same properties are set when launched by terraform. See https://github.com/openshift/installer/blob/bc8db97a005785e6524ff0e504d2c4d0c651dc94/upi/vsphere/machine/main.tf#L45-L50.

Reply:
@staebler yeah, I saw such code, but do you where can I review the vapp setting from vsphere client web console?

Reply:
> @staebler yeah, I saw such code, but do you where can I review the vapp setting from vsphere client web console?

If you set the vApp property after the clone (which is how terraform does it), then you can see the property under the Configure tab of the VM. See https://github.com/openshift/openshift-docs/pull/14523#discussion_r279769953. However, if you set the property using the Configuration Parameters, the values will not show up in the vApp properties on the Configure tab.

Reply:
But, to be clear, I have tested setting the vApp properties both ways (ie, from Configuration Parameters and from the Configure tab). It works both ways.

Reply:
![Screenshot_2019-04-30 vSphere - bootstrap-0 - vApp Options](https://user-images.githubusercontent.com/30120115/56976766-af52db80-6b41-11e9-958c-97492c53d234.png)

Reply:
![Screenshot_2019-04-30 vSphere - compute-0 - vApp Options](https://user-images.githubusercontent.com/30120115/56976893-f5a83a80-6b41-11e9-98ff-2d1013da783e.png)